Makefile

```diff
@@ -5,14 +5,10 @@ env:
 	workon up42-py
 
 install:
```

Comment:
1. pytest - to switch to poetry
2. e2e - to switch to poetry. Do we actually need this as a separate step?
3. serve - better to switch to poetry to make sure that it runs with the specified dependencies
4. gh-pages - poetry

Reply:
> Do we actually need this as a separate step?

Not in the scope of this PR

> serve - better to switch to poetry to make sure that it runs with the specified dependencies

> gh-pages - poetry

Couldn't find the equivalent poetry command. Can you please re-direct me to relevant documentation.